client/app/components/dynamic-table/default-cell/index.js

```diff
@@ -7,10 +9,35 @@ export default function init(ngModule) {
     replace: true,
     scope: {
       column: '=',
-      value: '=',
+      row: '=',
     },
     link: ($scope) => {
-      $scope.sanitize = value => $sanitize(value);
+      // `dynamicTable` will recreate all table cells if some columns changed.
+      // This means two things:
+      // 1. `column` object will be always "fresh" - no need to watch it.
+      // 2. we will always have a column object already available in `link` function.
+      // Note that `row` may change during this directive's lifetime.
+
+      if ($scope.column.displayAs === 'string') {
+        $scope.allowHTML = $scope.column.allowHTML;
+      } else {
+        $scope.allowHTML = contains(['image', 'link'], $scope.column.displayAs);
+      }
+
+      let renderValue;
+      switch ($scope.column.displayAs) {
+        case 'image': renderValue = renderImage; break;
+        case 'link': renderValue = renderLink; break;
+        default: renderValue = renderDefault; break;
+      }
```

Comment:
```javascript
const types = {image: renderImage, link: renderLink};
const renderValue = types[$scope.column.displayAs] || renderDefault;
```

How about this instead of a switch case?

Right now it's not a huge difference, but as the column types list grows it will become more manageable.

Reply:
:+1: